packages/containers/Bootstrap/Steps/MasterPasswordStep/MasterPasswordCheckForm.js

```diff
@@ -94,7 +95,7 @@ const MasterPasswordCheckForm = ({ currentUser, onSubmit }) => {
             name="password"
             value={values.password}
             autoFocus
-            maxLength={24}
+            maxLength={MAX_PASSWORD_LENGTH}
```

Comment:
I'm thinking.. Do we really need validation when we check MP? It just must be same with MP from prev step.

Reply:
@cu4xxi good point

Reply:
Removed validation